cmd/manager/main.go

```diff
@@ -101,9 +157,46 @@ func main() {
 	cl := mgr.GetClient()
 	catalogClient := catalogclient.New(cl, cache.NewFilesystemCache(cachePath, &http.Client{Timeout: 10 * time.Second}))
 
+	cfgGetter, err := helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), helmclient.StorageNamespaceMapper(func(o client.Object) (string, error) {
+		return systemNamespace, nil
+	}))
+	if err != nil {
+		setupLog.Error(err, "unable to config for creating helm client")
+		os.Exit(1)
+	}
+
+	acg, err := helmclient.NewActionClientGetter(cfgGetter)
+	if err != nil {
+		setupLog.Error(err, "unable to create helm client")
+		os.Exit(1)
+	}
+
+	unpacker, err := source.NewDefaultUnpacker(mgr, systemNamespace, unpackImage, (*x509.CertPool)(nil))
+	if err != nil {
+		setupLog.Error(err, "unable to create unpacker")
+		os.Exit(1)
+	}
+
+	storageURL, err := url.Parse(fmt.Sprintf("%s/bundles/", httpExternalAddr))
+	if err != nil {
+		setupLog.Error(err, "unable to parse bundle content server URL")
+		os.Exit(1)
+	}
+
+	localStorage := &storage.LocalDirectory{
+		RootDirectory: provisionerStorageDirectory,
+		URL:           *storageURL,
+	}
```

Comment:
Instead of having a storage method that stands up an HTTP server, I think it could make more sense to either:
1. Always unpack and use the returned contents (could be optimized with a cache in the future - especially since a direct image registry client was recently added to the rukpak library that uses a cache and there are plans to switch to this in a follow-up)
2. Still have a `Storage` interface, but write an implementation that doesn't stand up an HTTP server and essentially acts as a "cache" for the unpacked bundle contents.

Reply:
Removed the need of having an HTTP endpoint for now. We now unpack contents and store in `/var/cache/bundles` and load it from there. But this does relate to the comment [here](https://github.com/operator-framework/rukpak/pull/874#discussion_r1592945966) where we can have the direct image registry client write in a common location, no longer requiring a storage dir.